### ISSUE TITLE
无法Freeze或者Unfreeze App

### ISSUE THREAD

**Issue by lambdaphy** [oasisfeng/island#499]
手机：Pixel 6
系统版本：Android 14
Island版本：6.3

之前用都正常，但是今天突然发现Island中的App无法Freeze或者Unfreeze，提示”Island is not ready (probably deactivated)“。

前两天更新了24年3月份的安全更新，不知道是不是这个原因。

**Comment by kevinlyliu** [comment]
Yes. Oasis seems forget this project now, enjoying travel with family.
